### ISSUE TITLE
Channels not working on 3.2.2 (627)

### ISSUE THREAD

**Issue by helfio** [slp/Telegram-FOSS#44]
The apps says 'You were removed from this group', but on desktop I still can access to the channel.
![photo_2015-10-03_16-14-11](https://cloud.githubusercontent.com/assets/9520033/10263310/e21e2fb8-69e9-11e5-819c-7c41e562112a.jpg)
![photo_2015-10-03_16-14-09](https://cloud.githubusercontent.com/assets/9520033/10263311/e2219e0a-69e9-11e5-8dd4-5240ece65395.jpg)

**Comment by omyno** [comment]
I can confirm this. Also, trying to join a channel results in a screen showing "Loading..." indefinitely for me.

**Comment by slp** [comment]
It works for me in both the emulator and a CM based phone. Could you try uninstalling the app and reinstalling it again?

**Comment by helfio** [comment]
Fresh makes this work fine, but it shouldn't be necessary though.

**Comment by slp** [comment]
@helfio Yup, I agree with you. Did you ever have the official Telegram app installed on that phone?

**Comment by helfio** [comment]
Yeah: I had 3.1.3 FOSS, then the official 3.1.3, then back to 3.1.3 FOSS and then tried update to 3.2.2 FOSS and that was when I catch this error. I went back to 3.1.3 FOSS and now after a fresh install 3.2.2. FOSS. I cannot remember more history.

Do you think it's related?

**Comment by slp** [comment]
I'm not sure, but I have here what I call a "clean" phone, where I only install F-Droid releases (I don't even use for debugging my own builds, to avoid "contamination"), and I've never found such issues.

**Comment by Bubu** [comment]
@slp: I only ever installed the f-droid versions of Telegram. No foreign apks and no own builds. I still got this Problem aftter upgrading to the last two versions. Each time fixed with a fresh install form fdroid.

**Comment by valerio-bozzolan** [comment]
Also for me channels work only after a fresh install.

**Comment by omyno** [comment]
Will existing secret chats or any other data disappear when I uninstall and re-install Telegram?

**Comment by Bubu** [comment]
Secret chats unfortunately yes. All other data is synchronized with telegram servers.

**Comment by Bubu** [comment]
Basically a duplicate of #45
